mvc/src/main/java/nextstep/mvc/controller/tobe/AnnotationHandlerMapping.java

```diff
@@ -21,10 +28,48 @@ public AnnotationHandlerMapping(final Object... basePackage) {
     }
 
     public void initialize() {
+
         log.info("Initialized AnnotationHandlerMapping!");
+        findHandlerFromPackage();
     }
 
     public Object getHandler(final HttpServletRequest request) {
+        String requestUrl = request.getRequestURI();
+        RequestMethod requestMethod = RequestMethod.valueOf(request.getMethod());
+        HandlerKey handlerKey = new HandlerKey(requestUrl, requestMethod);
+        return handlerExecutions.get(handlerKey);
+    }
+
+    private void findHandlerFromPackage() {
+        for (Object eachPackage : basePackage) {
+            Reflections reflections = new Reflections(eachPackage);
+            Set<Class<?>> classes = reflections.getTypesAnnotatedWith(Controller.class);
+            findMethodFromClass(classes);
+        }
+    }
+
+    private void findMethodFromClass(Set<Class<?>> classes) {
+        for (Class<?> clazz : classes) {
+            Method[] methods = clazz.getMethods();
+            findRequestMappingFromMethod(methods);
+        }
+    }
+
+    private Method findRequestMappingFromMethod(Method[] methods) {
+        for (Method method: methods) {
+            addRequestMapping(method);
+        }
         return null;
```

Comment:
```suggestion
    private void findRequestMappingFromMethod(Method[] methods) {
        for (Method method: methods) {
            addRequestMapping(method);
        }
```
이건 어떨까요ㅎ.ㅎ
메서드명도 이에 맞춰 변경해주시면 좋을 것 같아요~

Reply:
stream을 사용하도록 전체적으로 로직을 변경하였습니다~